xml/bp_troubleshooting_status.xml

```diff
@@ -866,6 +866,100 @@
       </listitem>
     </varlistentry>
   </variablelist>
+  <variablelist><title>HEALTH CHECKS</title>
+  <varlistentry>
+    <term><option>CEPHADM_PAUSED</option></term>
+    <listitem>
+      <para>
+        &cephadm; background work has been paused with <command>ceph orch pause</command>.
+        &cephadm; continues to perform passive monitoring activities (like
+        checking host and daemon status), but it will not make any changes
+        (like deploying or removing daemons).
+      </para>
+      <para>
+        Resume &cephadm; work with:
+      </para>
+<screen>&prompt.cephuser;ceph orch resume</screen>
+    </listitem>
+  </varlistentry>
+  <varlistentry>
+    <term><option>CEPHADM_STRAY_HOST</option></term>
+    <listitem>
+      <para>
+        One or more hosts have running &ceph; daemons but are not registered
+        as hosts managed by &cephadm;. This means that those services cannot
+        currently be managed by &cephadm;. For example, restarted, upgraded, included
+        in <command>ceph orch ps</command>.
+      </para>
+      <para>
+        You can manage the host(s) with:
+      </para>
+<screen>&prompt.cephuser;ceph orch host add *<replaceable>hostname</replaceable>*</screen>
```

Comment:
Do the star signs '*' really belong here?

Reply:
My understanding is yes. But, @sebastian-philipp is that correct?

Reply:
```suggestion
<screen>&prompt.cephuser;ceph orch host add <replaceable>hostname</replaceable></screen>
```